packages/react/spec/useFetch.spec.ts

```diff
@@ -169,7 +169,28 @@ describe("useFetch", () => {
     expect(reexecuteResult).toEqual("fixed");
   });
 
-  test("it doesn't automatically start sending POST requests", async () => {
+  test("it automatically starts sending requests with no method specified", async () => {
```

Comment:
Maybe I missed it, but it look like there's no test to ensure it's a GET request when no method is specified. Wonder if we should add one / expectations?

Reply:
It is a good point -- right now we're actually inheriting that behavior from `fetch` itself which I think is the right move. I will add a test to ensure we pass no method and let it pick the default.